scripts/cmake/vcpkg_build_make.cmake

```
@@ -159,27 +228,44 @@ function(vcpkg_build_make)
                 endif()
             
                 message(STATUS "Installing ${TARGET_TRIPLET}${SHORT_BUILDTYPE}")
-                if (CMAKE_HOST_WIN32)
-                    # In windows we can remotely call make
-                    set(WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}${SHORT_BUILDTYPE})
-                    vcpkg_execute_build_process(
-                        COMMAND "${MAKE} ${INSTALL_OPTS}"
-                        WORKING_DIRECTORY ${WORKING_DIRECTORY}
-                        LOGNAME "install-${TARGET_TRIPLET}${SHORT_BUILDTYPE}"
-                    )
+                
+                if(MAKE_BASH)
+                    set(MAKE_CMD_LINE "${MAKE_COMMAND} ${INSTALL_OPTS}")
                 else()
-                    set(WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}${SHORT_BUILDTYPE}${_VCPKG_PROJECT_SUBPATH})
-                    vcpkg_execute_build_process(
-                        COMMAND "${MAKE};${INSTALL_OPTS}"
-                        WORKING_DIRECTORY ${WORKING_DIRECTORY}
-                        LOGNAME "install-${TARGET_TRIPLET}${SHORT_BUILDTYPE}"
-                    )
+                    set(MAKE_CMD_LINE ${MAKE_COMMAND} ${INSTALL_OPTS})
                 endif()
+                
+                set(WORKING_DIRECTORY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}${SHORT_BUILDTYPE}")
+                vcpkg_execute_build_process(
+                    COMMAND ${MAKE_BASH} ${MAKE_CMD_LINE}
+                    WORKING_DIRECTORY "${WORKING_DIRECTORY}"
+                    LOGNAME "install-${TARGET_TRIPLET}${SHORT_BUILDTYPE}"
+                )
+
             endif()
         endforeach()
+        string(REGEX REPLACE "([a-zA-Z]):/" "/\\1/" _VCPKG_INSTALL_PREFIX "${CURRENT_INSTALLED_DIR}")
+        file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}_tmp")
+        file(RENAME "${CURRENT_PACKAGES_DIR}" "${CURRENT_PACKAGES_DIR}_tmp")
+        file(RENAME "${CURRENT_PACKAGES_DIR}_tmp${_VCPKG_INSTALL_PREFIX}/" "${CURRENT_PACKAGES_DIR}")

```

?
```suggestion
        file(RENAME "${CURRENT_PACKAGES_DIR}_tmp" "${CURRENT_PACKAGES_DIR}")

```

  file RENAME failed to rename
    E:/tools/vcpkg/packages/gperf-tools_x64-windows_tmp/E/tools/vcpkg/installed/x64-windows/
  to
    E:/tools/vcpkg/packages/gperf-tools_x64-windows
  because: No such file or directory

---

A few details here why it is how it is: 
- prefix must be pointed to the final install location which is `CURRENT_INSTALLED_DIR`
- first "installation" must be into `CURRENT_PACKAGES_DIR` due to vcpkg rules how it stages installs
- this is achieved by using [DESTDIR](https://www.gnu.org/prep/standards/html_node/DESTDIR.html) and setting it to `CURRENT_PACKAGES_DIR`
- thus make install goes into ${CURRENT_PACKAGES_DIR}/${CURRENT_INSTALLED_DIR} which is equivalent to DESTDIR/PREFIX
- since VCPKG does not support such a folder hierarchy `${CURRENT_PACKAGES_DIR}/${CURRENT_INSTALLED_DIR}/<installed_folders>` must be renamed to `${CURRENT_PACKAGES_DIR}/<installed_folders>`


>file RENAME failed to rename
E:/tools/vcpkg/packages/gperf-tools_x64-windows_tmp/E/tools/vcpkg/installed/x64-windows/

This probably means nothing has been installed and might indicate that a manual installation is needed by calling `vcpkg_build_make` instead of `vcpkg_install_make`. Another possibility is that the makefile is not following GNU coding standards and installs into some other folder without `DESTDIR` support and should be patched to follow coding standards.

---

> DESTDIR=${_VCPKG_PACKAGE_PREFIX}) + "--bindir=${prefix_var}/tools/${PORT}/bin"
> =
> E:/tools/vcpkg/packages/gperf-tools_x64-windows_tmp/E/tools/vcpkg/installed/x64-windows/
> rename to
>  E:/tools/vcpkg/packages/gperf-tools_x64-windows


accessibly